packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -254,4 +266,19 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
     </SafeAreaView>
     );
   }
+
+  private _saveFormState = (): void => {
+    FeedbackForm._savedState = { ...this.state };
+  };
```

Comment:
Would it work if we only save it when closing the form instead of on every state change? Just to avoid the object recreation potentially on every keystroke when typing.

Reply:
That's a good idea @krystofwoldrich 👍 
Updated 59f2f570ae0e6bb6744bb59566cdc13e549add75.